src/pages/settings/Payments/PaymentsPage.js

```diff
@@ -121,9 +131,19 @@ class PaymentsPage extends React.Component {
                         onCloseButtonPress={() => Navigation.dismissModal(true)}
                     />
                     <ScrollView style={styles.flex1}>
-                        {
-                            Permissions.canUseWallet(this.props.betas) && <CurrentWalletBalance />
-                        }
+                        {Permissions.canUseWallet(this.props.betas) && (
```

Comment:
Just to double check, regular users aren't on this beta right? So this button won't show for them? Just want to make sure we're not leading people to a blank page

Reply:
I am not sure about it yet but I can remove this until the next PR.

Reply:
All of `expensify.com` and a handful of testers are on the beta.

Reply:
so I should remove it for now.

Reply:
FWIW this is what someone will see if they are not on the beta

![2021-12-21_10-31-41](https://user-images.githubusercontent.com/32969087/146993750-602d6ad8-c1b8-43f5-ad8b-6d0b851e8169.png)

So the Transfer Balance button won't appear but they do see a weird message and are able to access the "Payments" screen via "Settings"

![2021-12-21_10-32-54](https://user-images.githubusercontent.com/32969087/146993895-f1137a82-d234-480b-827d-3ab35b5d2f2b.png)

Reply:
>so I should remove it for now.

I think we could have just left it, but it's fine to add it in another PR